### PR TITLE
[kr] Korean translation for lesson 4 connect-internet of 1-getting-started

### PR DESCRIPTION
**Pull request by DobiIsFree**
This is korean translation for lesson 4 connect-internet of chapter 1.getting-started.

- README.ko.md
- assignment.ko.md

### CODE REVIEW

**Comment by DobiIsFree** [comment]
@microsoft-github-policy-service agree

File: 1-getting-started/lessons/4-connect-internet/translations/README.ko.md [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+
+> 🎓 센서에서 수집되어 클라우드로 전송되는 데이터를 telemetry라고 합니다.
+
+IoT 장치는 클라우드에서 메시지를 수신할 수 있습니다. 종종 메시지에는 다음 명령들이 포함되어 있습니다. - 내부적으로 작업 수행하기(예: 재부팅 또는 펌웨어 업데이트) 또는 엑추에이터 사용하기(예: 조명 켜기).
+
+이 강의에서는 IoT 장치가 클라우드에 연결하는데 사용할 수 있는 몇가지 통신 프로토콜과 송수신할 수 있는 데이터 유형을 소개합니다. 또한 야간 조명에 인터넷 제어를 추가하고, LED 제어 논리를 로컬에서 실행되는 ‘서버’ 코드로 이동하는 실습을 하게 됩니다.
+
+이 강의에서 다룰 내용은 다음과 같습니다:
+
+- [통신 프로토콜](notion://www.notion.so/chap-1_4-0ec7f5df23cf4754b02624ce55051dce#communication-protocols)
```
Comment (jhYun505) [inline 1-getting-started/lessons/4-connect-internet/translations/README.ko.md:31]: I think you need to modify the connected link. It is connected to notion.

File: 1-getting-started/lessons/4-connect-internet/translations/README.ko.md [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+
+MQTT 브로커에게 가벼운 수준의 원격 분석을 보냅니다.
+
+데이터는 키/값 쌍을 사용해 텍스트로 데이터를 인코딩하는 표준인 JSON(JavaScriopt Object Notation의 약자)으로 인코딩되어 전송됩니다.
+
+✅ 이전에 JSON을 접한 적이 없다면, [JSON.org documentation](https://www.json.org/)에서 더 자세히 알아볼 수 있습니다.
+
+장치에서 MQTT 브로커로 원격 분석을 보내려면 아래 단계를 따르십시오:
+
+- [아두이노 - Wio 터미널](notion://www.notion.so/wio-terminal-telemetry.md)
```
Comment (jhYun505) [inline 1-getting-started/lessons/4-connect-internet/translations/README.ko.md:137]: Same in here. It is also notion link.

File: 1-getting-started/lessons/4-connect-internet/translations/single-board-computer-commands.ko.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+   MQTT 클라이언트는 서버가 메시지를 보낼 주제를 실행하고 메시지가 수신될 때 `handle_command` 함수가 호출되도록 설정합니다.
+
+   > 💁 `on_message` 핸들러는 실행된 모든 주제에 대해 호출됩니다. 나중에 여러 주제를 수신하는 코드를 작성하면, 핸들러 함수를 거쳐 보내진 `message` 객체의 주제를 얻을 수 있습니다.
+
+5. 과제의 이전 부분과 동일한 방식으로 코드를 실행합니다. 가상 IoT 장치를 사용하는 경우, CounterFit 앱이 실행중이고 LED가 올바른 핀에 생성되었는지 확인하십시오.
+
+6. 물리적 또는 가상 장치에서 감지한 조명 수준을 조정합니다. 수신 중인 메시지와 전송 중인 명령이 터미널에 기록됩니다. LED도 조명 수준에 따라 켜지고 꺼질 것입니다.
+
+> 💁 이 코드는 [code-commands/virtual-device](code-commands/virtual-device) 폴더 또는 [code-commands/pi](code-commands/pi) 폴더에서 찾을 수 있습니다.
```
Comment (jhYun505) [inline 1-getting-started/lessons/4-connect-internet/translations/single-board-computer-commands.ko.md:51]: I think you should change the links like `code-commands/virtual-device` -> `../code-commands/virtual-device`.

File: 1-getting-started/lessons/4-connect-internet/translations/README.ko.md [hunk 4]
```diff
@@ -0,0 +1,472 @@
...
+>
+> ```
+> listener 1883 0.0.0.0
+> allow_anonymous true
+>
+> ```
+
+## 과제
+
+[MQTT를 다른 통신 프로토콜과 비교 및 대조](notion://www.notion.so/assignment.md)
```
Comment (jhYun505) [inline 1-getting-started/lessons/4-connect-internet/translations/README.ko.md:472]: Also in here. Change the link

**Review by jhYun505 — CHANGES_REQUESTED** [review]
I think you need to check some links.

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**